pkg/api/common/helper.go

```diff
@@ -103,6 +103,8 @@ func IsNvidiaEnabledSKU(vmSize string) bool {
 		"Standard_NC12s_v3":  true,
 		"Standard_NC24s_v3":  true,
 		"Standard_NC24rs_v3": true,
+		"Standard_ND40s_v3":  true,
+		"Standard_ND40rs_v2": true,
```

Comment:
Standard_ND40rs_v2 used the Tesla P40 GPU, it should be on line 96
I don't know about Standard_ND40s_v3 (it's not listed in the docs but I assume it's also a P40 GPU since it's part of the ND series).

source https://docs.microsoft.com/en-us/azure/virtual-machines/linux/sizes-gpu

Reply:
no they use V100 as per documentation : 
https://docs.microsoft.com/en-us/azure/virtual-machines/linux/sizes-gpu#ndv2-series-preview

Reply:
and correct about the not listed one, it's in preview so not really fixed